crates/containerd-shim-wasm/src/sandbox/oci.rs

```diff
@@ -7,10 +7,23 @@ use std::os::unix::process::CommandExt;
 use std::process;
 
 use anyhow::Context;
+use oci_spec::image::Descriptor;
 pub use oci_spec::runtime::Spec;
 
 use super::error::Result;
 
+pub const COMPONENT_ARTIFACT_TYPE: &str = "application/vnd.bytecodealliance.component.v1+wasm";
+pub const MODULE_ARTIFACT_TYPE: &str = "application/vnd.bytecodealliance.module.v1+wasm";
```

Comment:
✔️

Reply:
This will be removed.  The images produced with wasm module/component layers will not be considered artifacts since for now they use the `config.mediatype` of `application/vnd.oci.image.config.v1+json` to be compatible with containerd, see dissussion in https://github.com/opencontainers/image-spec/pull/1137 and https://github.com/opencontainers/image-spec/pull/1141